infra/gcp-broad/main.tf

```diff
@@ -490,6 +487,23 @@ resource "google_storage_bucket_iam_member" "batch_hail_query_bucket_storage_vie
   member = "serviceAccount:${module.batch_gsa_secret.email}"
 }
 
+module "test_batch_gsa_secret" {
+  source = "./gsa"
+  name = "test-batch"
+  project = var.gcp_project
+  iam_roles = [
+    "compute.instanceAdmin.v1",
```

Comment:
totally unrelated to this PR, but at some point we should scope this to the test namespaced VMs.